constants/chatbotCommands.py

```diff
@@ -2793,3 +2793,32 @@ async def disable_profiling(fro: str, chan: str, message: list[str]) -> str:
     profiling.profiler.disable()
     profiling.profiler.dump_stats("stats.dump")  # TODO: persist in s3
     return "Profiler disabled. Dumped stats to disk @ stats.dump"
+
+
+@command(trigger="!leaderboardsize", syntax="<size>", hidden=True)
```

Comment:
```suggestion
@command(trigger="!leaderboardsize", syntax="<size>")
```
i don't think hiding the command is necessary

Reply:
nvm, i misunderstood what hidden is for. keep it